internal/services/services.go

```diff
@@ -163,10 +165,27 @@ func printProxyURL(w io.Writer) error {
 		return errors.WithStack(err)
 	}
 
-	color.New(color.FgHiGreen).Fprintf(
-		w,
-		"To access services on this vm use: %s-<port>.svc.devbox.sh\n",
-		hostname,
-	)
+	printGeneric := false
+	for _, service := range services {
+		if port, _ := service.Port(); port != "" {
+			color.New(color.FgHiGreen).Fprintf(
+				w,
+				"To access %s on this vm use: %s-%s.svc.devbox.sh\n",
+				service.Name,
+				hostname,
+				port,
+			)
+		} else {
+			printGeneric = true
+		}
+	}
+
+	if printGeneric {
+		color.New(color.FgHiGreen).Fprintf(
+			w,
+			"To access other services on this vm use: %s-<port>.svc.devbox.sh\n",
+			hostname,
+		)
+	}
```

Comment:
what if I have 3 services, but only one has a port, then this will print for just the one having-port-service. That seems problematic because users will worry about how to access the other-non-explicity-port-services.

Could we improve this to:
```
To access <having-port-service> on this vm use: ....
To access other services on this vm use: .....
```

Reply:
I think the current behavior is what you are describing. If at least one service doesn't have a declared port, `printGeneric` will be `true` so we print the last message.

Reply:
oh you are right. booleans are hard!